hack/Dockerfile

```diff
@@ -141,13 +134,11 @@ RUN set -ex \
 		/src/src/dashboard/src/media \
 	' \
 	&& mkdir -p $internalDirs \
-	&& chown -R archivematica $internalDirs
+	&& chown -R archivematica $internalDirs \
+    && npm install -g npm@9.8.1 /src/src/dashboard/frontend
```

Comment:
I presume this has the same effect of the `yarn ... install ...` line below? If that's the case the output of this command will be owned by `root` and not by the `archivematica` user.

Reply:
Thanks for suggestions. I have installed npm globally by root user separately.